pkg/secrethub/dir.go

```diff
@@ -169,6 +174,35 @@ func (s dirService) Delete(path string) error {
 	return nil
 }
 
+// CreateAll creates all directories in the given path that do not exist yet.
+func (s dirService) CreateAll(path string) error {
+	err := api.ValidateDirPath(path)
+	if err != nil {
+		return err
+	}
+	return s.createAll(path)
+}
+
+func (s dirService) createAll(path string) error {
+	if len(strings.Split(path, "/")) < 3 {
+		return nil
+	}
+
+	_, err := s.GetTree(path, 0, false)
+	if err != api.ErrDirNotFound {
+		return err
```

Comment:
When the dir already exists, doesn't this return an error? Oh wait, then it returns `nil`, correct?

Consider adding an explanatory comment here because it's not suuuper clear at a glance.